bigtop-deploy/puppet/manifests/python.pp

```diff
@@ -20,4 +20,12 @@
           target => '/usr/bin/python2',
         }
     }
+    
+    # BIGTOP-4069 create the soft link of python for openEuler smoke-tests
+    if (($operatingsystem == 'openEuler' )) {
+        file { '/usr/bin/python':
+          ensure => 'link',
+          target => '/usr/bin/python3',
+        }
+    }
```

Comment:
python3-unversioned-command seems to be provided for the purpose in openEuler too. How about installing it? @MacChen01

```
[root@929191cea27f /]# dnf install python3-unversioned-command

[root@929191cea27f /]# rpm -q -f /usr/bin/python
python3-unversioned-command-3.9.9-28.oe2203sp3.x86_64

[root@929191cea27f /]# ls -lh /usr/bin/python
lrwxrwxrwx 1 root root 16 Dec 28 05:49 /usr/bin/python -> /usr/bin/python3
```

Reply:
> python3-unversioned-command seems to be provided for the purpose in openEuler too. How about installing it? @MacChen01
> 
> ```
> [root@929191cea27f /]# dnf install python3-unversioned-command
> 
> [root@929191cea27f /]# rpm -q -f /usr/bin/python
> python3-unversioned-command-3.9.9-28.oe2203sp3.x86_64
> 
> [root@929191cea27f /]# ls -lh /usr/bin/python
> lrwxrwxrwx 1 root root 16 Dec 28 05:49 /usr/bin/python -> /usr/bin/python3
> ```
@iwasakims 
I think using symbolic links to ensure all systems have /usr/bin/python -> /usr/bin/python3 in packages.pp offers stronger universality because not all systems support python3-unversioned-command. This is also the approach I took in BIGTOP-4098.
![image](https://github.com/apache/bigtop/assets/18082602/c18f73e6-3ec1-4b0d-969c-87bfbfcf51ba)

Reply:
> stronger universality because not all systems support python3-unversioned-command

We don't need to worry about other distros by conditional like `if (($operatingsystem == 'openEuler' )) {`. We usually avoid overwriting the file provided by standard packages of the OS distribution.

Reply:
> overwriting

ok, thanks